identity-service/sequelize/migrations/20200506213609-remix-notification.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
```

Comment:
wow intense migration

Reply:
Yeah, removing values from a num is kinda hard.